### DISCUSSION TITLE
Is there anyway to set default value to the media upload?

### DISCUSSION THREAD

**Discussion by Duckinm** [payloadcms/payload#278] [Q&A]
.

**Comment by jmikrut** [comment]
Hey @Duckinm — there sure is!

You can set an `upload` field's `defaultValue` to the ID of an upload and everything will work as expected. The catch is that you'll need to know that ID before runtime, meaning you might need to write a seed script ([example here](https://github.com/payloadcms/nextjs-custom-server/blob/master/seed/index.js)) which populates your database with an upload and then writes its ID to a `json` file somewhere in your project. That `json` file would likely be ignored from your repo and would need to be run both locally and in your deployment environment(s) before booting up Payload for the first time.

Then, from there, you'd simply import that `json` file in your Payload config and use the ID of your newly seeded upload as the default value of your upload field.

Or you could do it in a `beforeChange` field hook but that would mean that the admin UI would not display a `defaultValue` until the document is saved. Then it'd all work normally. 

A few options, really. What do you think?
